components/Chat/Message/Message.tsx

```diff
@@ -133,9 +145,55 @@ const MessageSenderAvatar = ({ message }: { message: MessageToDisplay }) => {
   );
 };
 
-function ChatMessage({ message, colorScheme, isGroup, isFrame }: Props) {
+const ChatMessage = ({
+  account,
+  message,
+  colorScheme,
+  isGroup,
+  isFrame,
+  showTime,
+  toggleTime,
+}: Props & { showTime: boolean; toggleTime: () => void }) => {
```

Comment:
we don't need to pass showTIme & toggleTime, let's define them in that component directly and remove them from the cached component below?